### PR TITLE
Add 2020 SLT CDT HPC presentation

### PR DESCRIPTION
**Pull request by willfurnass**

### CODE REVIEW

File: _posts/2020-09-22-slt-cdt-hpc-pres.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: "Introducing HPC to new PhD students"
+author: Will Furnass
+slug: 2020-09-22-slt-cdt-hpc-pres
+date: 2020-09-22 12:00:00 UTC
+tags: 
+category:
+link:
+description:
```
Comment (drj11) [inline _posts/2020-09-22-slt-cdt-hpc-pres.md:10]: Can you add an image for the social card?

**Review by drj11 — APPROVED** [review]
good!

File: _posts/2020-09-22-slt-cdt-hpc-pres.md (hunk 1 above)
Comment (willfurnass) [reply _posts/2020-09-22-slt-cdt-hpc-pres.md:10]: I've done this.  Know if there are any constraints on image type/size?

Comment (drj11) [reply _posts/2020-09-22-slt-cdt-hpc-pres.md:10]: i have no idea.

Comment (willfurnass) [reply _posts/2020-09-22-slt-cdt-hpc-pres.md:10]: @davidwilby Do you know?

Comment (willfurnass) [reply _posts/2020-09-22-slt-cdt-hpc-pres.md:10]: @drj11 Thanks for the review.

**Pull request merged**